ts/src/phemex.ts

```diff
@@ -3434,75 +3434,75 @@ export default class phemex extends Exchange {
 
     parsePosition (position, market = undefined) {
         //
-        //   {
-        //     userID: '811370',
-        //     accountID: '8113700002',
-        //     symbol: 'ETHUSD',
-        //     currency: 'USD',
-        //     side: 'Buy',
-        //     positionStatus: 'Normal',
-        //     crossMargin: false,
-        //     leverageEr: '200000000',
-        //     leverage: '2.00000000',
-        //     initMarginReqEr: '50000000',
-        //     initMarginReq: '0.50000000',
-        //     maintMarginReqEr: '1000000',
-        //     maintMarginReq: '0.01000000',
-        //     riskLimitEv: '5000000000',
-        //     riskLimit: '500000.00000000',
-        //     size: '1',
-        //     value: '22.22370000',
-        //     valueEv: '222237',
-        //     avgEntryPriceEp: '44447400',
-        //     avgEntryPrice: '4444.74000000',
-        //     posCostEv: '111202',
-        //     posCost: '11.12020000',
-        //     assignedPosBalanceEv: '111202',
-        //     assignedPosBalance: '11.12020000',
-        //     bankruptCommEv: '84',
-        //     bankruptComm: '0.00840000',
-        //     bankruptPriceEp: '22224000',
-        //     bankruptPrice: '2222.40000000',
-        //     positionMarginEv: '111118',
-        //     positionMargin: '11.11180000',
-        //     liquidationPriceEp: '22669000',
-        //     liquidationPrice: '2266.90000000',
-        //     deleveragePercentileEr: '0',
-        //     deleveragePercentile: '0E-8',
-        //     buyValueToCostEr: '50112500',
-        //     buyValueToCost: '0.50112500',
-        //     sellValueToCostEr: '50187500',
-        //     sellValueToCost: '0.50187500',
-        //     markPriceEp: '31332499',
-        //     markPrice: '3133.24990000',
-        //     markValueEv: '0',
-        //     markValue: null,
-        //     unRealisedPosLossEv: '0',
-        //     unRealisedPosLoss: null,
-        //     estimatedOrdLossEv: '0',
-        //     estimatedOrdLoss: '0E-8',
-        //     usedBalanceEv: '111202',
-        //     usedBalance: '11.12020000',
-        //     takeProfitEp: '0',
-        //     takeProfit: null,
-        //     stopLossEp: '0',
-        //     stopLoss: null,
-        //     cumClosedPnlEv: '-1546',
-        //     cumFundingFeeEv: '1605',
-        //     cumTransactFeeEv: '8438',
-        //     realisedPnlEv: '0',
-        //     realisedPnl: null,
-        //     cumRealisedPnlEv: '0',
-        //     cumRealisedPnl: null,
-        //     transactTimeNs: '1641571200001885324',
-        //     takerFeeRateEr: '0',
-        //     makerFeeRateEr: '0',
-        //     term: '6',
-        //     lastTermEndTimeNs: '1607711882505745356',
-        //     lastFundingTimeNs: '1641571200000000000',
-        //     curTermRealisedPnlEv: '-1567',
-        //     execSeq: '12112761561'
-        //   }
+        //    {
```

Comment:
@samgermain also if possible let's avoid re-formatting comments because it adds to the noise in PRs like this one